extensions/cpsection/aboutme/view.py

```diff
@@ -418,5 +418,6 @@ def __gender_changed_cb(self, genderpicker, gender):
         self._age_pickers.update_gender(gender)
         return False
 
-    def __age_changed_cb(self, event, age):
+    @staticmethod
```

Comment:
Why make this method a static method?

Reply:
parameter `self` is passed, but not used.